app/models/hmcts_common_platform/case_summary.rb

```diff
@@ -0,0 +1,29 @@
+module HmctsCommonPlatform
+  class CaseSummary
+    attr_reader :data
+
+    def initialize(data)
+      @data = HashWithIndifferentAccess.new(data || {})
+    end
+
+    def case_status
+      data[:caseStatus]
+    end
+
+    def prosecution_case_id
+      data[:prosecutionCaseId]
+    end
+
+    def prosecution_case_reference
+      data[:prosecutionCaseReference]
+    end
+
+    def as_json
+      {
+        case_status:,
+        prosecution_case_id:,
+        prosecution_case_reference:,
+      }
```

Comment:
are you using a different approach respect the everywhere-used `def to_json`  ?

I wonder if we should stick with the exing pattern, for consistency I mean

Reply:
Sigh, you're right. I don't like using `to_json` and having it return a Hash instead of a JSON string, but it's better to be consistent. I'll fix it.